src/static/current-state/components/tabs/SystemTab.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import { TabContent } from './TabContent';
+import { User } from '../User';
+import { PermissionControl } from '../PermissionControl';
+
+// sort by permssion descending (admin -> land -> read)
+function sortUsersByPermission(user1: IPermission, user2: IPermission) {
+  const permssionsLevels = ['read', 'land', 'admin'];
+  return permssionsLevels.indexOf(user2.mode) - permssionsLevels.indexOf(user1.mode);
+}
+
+export type SystemTabProps = {
+  allowedUsers: IPermission[];
+  loggedInUser: ISessionUser;
+  defaultPaused: boolean;
+};
+
+export type SystemTabsState = {
+  paused: boolean;
+};
+
+export class SystemTab extends React.Component<SystemTabProps, SystemTabsState> {
+  constructor(props: SystemTabProps) {
+    super(props);
+    this.state = {
+      paused: props.defaultPaused,
+    };
+  }
+
+  onPauseChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const checked = e.target.checked;
+    if (checked) {
+      const reason = window.prompt(
+        'What would you like the pause message to be?',
+        'Builds have been paused by an admin, see the Fabric Build room for details',
+      );
+      if (reason === null) return;
+      fetch('/api/pause', {
+        method: 'POST',
+        headers: new Headers({ 'Content-Type': 'application/json' }),
+        body: JSON.stringify({ reason }),
+      }).then(() => {
+        this.setState({ paused: true });
+        location.reload();
+      });
+    } else {
+      fetch('/api/unpause', { method: 'POST' }).then(() => {
+        this.setState({ paused: false });
+        location.reload();
+      });
+    }
+  };
+
+  onNextClick = () => {
+    fetch('/api/next', { method: 'POST' })
+      .then(r => r.json())
```

Comment:
Same here, this new API has a failure state.  The UI shouldn't be inconsistent with the returned status code